src/Aks/Aks/Commands/ImportAzureRmAksCredential.cs

```diff
@@ -243,6 +257,10 @@ public static string MergeKubeConfig(string original, string additions)
 
         private static YamlSequenceNode MergeNamedItems(YamlMappingNode original, YamlMappingNode addition, string key)
         {
+            if (!(original[new YamlScalarNode(key)] is YamlSequenceNode))
+            {
+                new Exception("Please make sure the format of your existing config file is following the standard. See the reference here: https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/");
```

Comment:
```suggestion
                throw new Exception("Please make sure the format of your existing config file is following the standard. See the reference here: https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/");
```
Missing `throw`. ALso please use the `AzPS***Exception` types defined in common code.